PlayCEA-API/src/PlayCEAClient.ts

```diff
@@ -661,10 +661,12 @@ class PlayCEAClient {
                 teamId: team.tid
             })),
             id: apiTournament.tmid,
-            brackets: apiTournament.bs.map(stage => ({
-                bracketId: stage.bid,
-                name: stage.name
-            })),
+            brackets: apiTournament.bs
+                .filter(stage => stage.bid === apiTournament.sbkt.reg || stage.bid === apiTournament.sbkt.po)
```

Comment:
This is filtering to only the live brackets rather than all placeholder/empty/hidden brackets.